okta/connect.go

```diff
@@ -128,3 +129,83 @@ func Connect(ctx context.Context, d *plugin.QueryData) (*okta.Client, error) {
 
 	return client, err
 }
+
+
+func ConnectV4(ctx context.Context, d *plugin.QueryData) (*oktaV4.APIClient, error) {
```

Comment:
Can we use the existing Connect function and update it with oktav4?

Reply:
@bigdatasourav, I believe that would be a separate PR, for migrating the V1 API to V4. Should we do it in this PR?